markdown/dev/reference/api/stack/generateTransform/en.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Stack.generateTransform()
+---
+
+The `Stack.generateTransform()` method generates transforms for the stack,
```

Comment:
let's say "generates SVG transforms for the stack"

Reply:
Added.